shell/client/shell.html

```diff
@@ -950,6 +956,7 @@ <h4 class="share-with-others">Share with others</h4>
 
 <template name="adminStats">
   <div id="stats">
+    <label><input class="enableStatsCollection" type="checkbox" name="reportStats" checked={{reportStats}}>Report anonymized stats below to sandstorm.io</label>
```

Comment:
I'd like to present a yes/no decision here, where they must click on one for the notification to disappear.

I just sent an e-mail to the mailing list to propose text and see what people think.